File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        boolean isDone = false;
        for(int i = 0; i < 3; i++) {
            String S = br.readLine();
            if(!isDone && !S.contains("Fizz") && !S.contains("Buzz")) {
                isDone = true;
                int num = Integer.parseInt(S) + (3 - i);
                if(num % 15 == 0) sb.append("FizzBuzz");
                else if(num % 5 == 0) sb.append("Buzz");
                else if(num % 3 == 0) sb.append("Fizz");
                else sb.append(num);
            }
        }
        System.out.println(sb.toString().trim());
    }
}

File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n').map(line => line.trim());
let isDone = false;
for(let i = 0; i < 3; i++) {
  if(!isDone && !input[i].startsWith("Fizz") && !input[i].startsWith("Buzz")) {
    isDone = true;
    let num = Number(input[i]) + (3 - i);
    if(num % 15 === 0) console.log("FizzBuzz");
    else if(num % 5 === 0) console.log("Buzz");
    else if(num % 3 === 0) console.log("Fizz");
    else console.log(num);
  }
}